### PR TITLE
32 epfdamage liquefaction and LS name fixes. 

### PR DESCRIPTION
**Pull request by vnarah2**
See the #32 for more details of the task. It handles the below:
1. Fix the broken liquefaction logic on EPF Damage
2. Add test for liquefaction, remove "refactored" test because they are not needed any more
3. Remove deprecated LS/DS name specific method
4. Use 10 digit precision when adjusting PGD limit states

Use the test_epfddamage.py to test

### CODE REVIEW

**Comment by vnarah2** [comment]
@ncsa-mo we need to create an issue to update the documentation related to these spec changes (liquefaction_fragility_key addition etc.) . Here: https://incore-dev.ncsa.illinois.edu/doc/incore/analyses/epf_dmg.html

**Review by navarroc — CHANGES_REQUESTED** [review]
Unused import on line 7, unused variables around line 57, extra space on line 290

**Comment by navarroc** [comment]
Other than the few cosmetic things, this change looks good and I tested it with a notebook I had for epf_damage and verified with/without liquefaction works, setting the liquefaction key works as well.

**Comment by navarroc** [comment]
We probably need to update the documentation to include the new liq_fragility_key.

**Review by longshuicy — APPROVED** [review]
LGTM approving

File: pyincore/analyses/epfdamage/epfdamage.py [hunk 1]
```diff
@@ -205,87 +257,28 @@ def epf_damage_analysis_bulk_input(self, epfs, hazard_type, hazard_dataset_id,
...
+                    damage_result['liqdemandunits'] = None
+                    damage_result['liqhazval'] = None
+                    damage_result['liqprobability'] = None
 
             ds_results.append(ds_result)
             damage_results.append(damage_result)
 
             i += 1
 
         #############################################################
```
Comment (navarroc) [inline pyincore/analyses/epfdamage/epfdamage.py:281]: Leftover comment line can probably be removed

**Review by navarroc — APPROVED** [review]
Besides the empty comment line, this looks good, approving

**Pull request merged**